src/core/openthread-core-windows-config.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I think it should go in examples/...  If the posix or cc2538 examples needed to change values found in the default-config, they should likewise have a config file.